pkg/util/flagext/bytesize_test.go

```diff
@@ -102,3 +104,33 @@ func Test_ByteSizeYAML(t *testing.T) {
 		})
 	}
 }
+
+func Test_ByteSizeJSON(t *testing.T) {
+	for _, tc := range []struct {
+		in  string
+		err bool
+		out ByteSize
+	}{
+		{
+			in:  `{ "bytes": "256GB" }`,
```

Comment:
Can you add a test case that mirrors the issue we had?

`2.62144e+07`

Reply:
Done.